client/src/pages/error.js

```diff
@@ -0,0 +1,22 @@
+import { Component } from 'preact'
+import config from '../config'
+import 'linkstate/polyfill'
+import withStyles from '../components/jss'
+
+export default withStyles({
+}, class Error extends Component {
+  componentDidMount () {
+    document.title = '404 Not Found' + config.ctfTitle
```

Comment:
A generic error component should probably support titles other than `404 Not Found`.